internal/zinc-compile-core/src/main/scala/sbt/internal/inc/AnalyzingCompiler.scala

```diff
@@ -177,12 +179,19 @@ final class AnalyzingCompiler(
     val arguments =
       compArgs.makeArguments(Nil, classpath, Some(outputDirectory), options)
     onArgsHandler(arguments)
-    call("xsbt.ScaladocInterface", "run", log)(
+    val (bridge, bridgeClass) = bridgeInstance(scaladocBridgeClassName, log)
+    bridge match {
+      case intf: ScaladocInterface2 =>
+        intf.run(sources.toArray, arguments.toArray[String], log, reporter)
+    }
+    /*
+      call("xsbt.ScaladocInterface", "run", log)(
```

Comment:
Need this to continue to support current sbt-dotty, no?

Reply:
Looks like this whole code is dead.

Reply:
Yea. Looks like it's missing the fallback.